### PR TITLE
Add homework #1 by Mykola Kulinich

### PR DESCRIPTION
**Pull request by NicolasKul**
I have 4 _".sh"_ files that you could test
Feel free to contact me if you have some questions

### CODE REVIEW

File: general/1_terminal/mykola-kulinich/t1.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#1/bin/zsh
```
Comment (sae3023) [inline general/1_terminal/mykola-kulinich/t1.sh:1]:
```suggestion
#!/bin/zsh
```

File: general/1_terminal/mykola-kulinich/t1.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+touch zo.txt zoo.txt zoola.txt zoolat.txt zoolatech.txt | ls | grep zoola
```
Comment (sae3023) [inline general/1_terminal/mykola-kulinich/t1.sh:3]: Let's add a cleanup section to remove the files that we have created.

File: general/1_terminal/mykola-kulinich/t4.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/bin/zsh
+
+jq '.messages[] | .text' $1 > t4.txt
```
Comment (sae3023) [inline general/1_terminal/mykola-kulinich/t4.sh:3]: I'll accept this, but if you'd like an additional assignment, try to make it look prettier.

Currently, it outputs arrays when it's a forward, mention or a link.

**Review by sae3023 — APPROVED** [review]

**Comment by sae3023** [comment]
Please fix the merge conflict.

**Comment by sae3023** [comment]
Let's create a clean up section in script 1 and merge.

**Comment by NicolasKul** [comment]
> 

Should I delete created files?

**Comment by sae3023** [comment]
> >  
> Should I delete created files?

Yes, the ones created by the `touch` command.

**Review by sae3023 — APPROVED** [review]

**Pull request merged**